app/Console/Commands/createSuperuser.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
We probably don't need this command as we can create a user via a seeder during migration e.g. `php artisan migrate:fresh --seed`.  We can remove this down the track though so no need to change the PR